### PR TITLE
Ports in /tg/ vending.dmi, additonal re-spriting.

### PR DESCRIPTION
**Pull request by Ghommie**
First step of many, in order to respect those who made the original sprites for their own codebase.

Changes: Many vendors sprites, where possible, now use /TG/Station's counterparts (slight modifications on wallmeds and secvendors).
For the ones without a counterpart, There are some new sprites. (or if you want, placeholders)
Plus made a slight variant sprite for the cargo munition vendor to distinguishing it from the armaments one.
And lot of unused sprites for the time being.

Open to any thought and review anyhow. Things may change in the future

### CODE REVIEW

**Comment by francinum** [comment]
Pictures of new sprites, please. Seems the iconbot died or something.

**Comment by Ghommie** [comment]
Iconbot seems to be fine now, it's probably just slown down somehow, but anyway [here some sprites at work](https://imgur.com/a/SF73a0y).

File: code/game/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -49,6 +49,14 @@
 
 	var/list/listed_products
 
+/obj/machinery/marine_selector/power_change()
+	..()
+	if( !(stat & NOPOWER) )
+		icon_state = initial(icon_state)
+	else
+		spawn(rand(0, 15))
+			src.icon_state = "[initial(icon_state)]-off"
```
Comment (ninjanomnom) [inline code/game/machinery/vending/new_marine_vendors.dm]: Why a spawn?
Also `src.` is not needed

Comment (Ghommie) [reply code/game/machinery/vending/new_marine_vendors.dm]: Basically this is copied from vending template, I suppose the `spawn` was added to make the row of vendors in the preps not go off in unison, an aestethic choice from the days where the marine vendors were a subtype of vending. But yes, I'll remove the `src.` from both .dm.

File: code/game/machinery/vending/marine_vending.dm [hunk 2]
```diff
@@ -231,8 +234,8 @@
 /obj/machinery/vending/marineFood
 	name = "\improper Marine Food and Drinks Vendor"
 	desc = "Standard Issue Food and Drinks Vendor, containing standard military food and drinks."
-	icon_state = "generic"
-	icon_deny = "generic-deny"
+	icon_state = "sustenance"
+	//icon_deny = "sustenance" //this thing doesn't even have restrictions
```
Comment (ninjanomnom) [inline code/game/machinery/vending/marine_vending.dm]: just remove the line

File: code/game/machinery/vending/new_marine_vendors.dm [hunk 3]
```diff
@@ -49,6 +49,14 @@
 
 	var/list/listed_products
 
+/obj/machinery/marine_selector/power_change()
+	..()
```
Comment (ninjanomnom) [inline code/game/machinery/vending/new_marine_vendors.dm]: `. = ..()`

**Pull request merged**